nifi-nar-bundles/nifi-smb-bundle/nifi-smb-processors/src/main/java/org/apache/nifi/processors/smb/ListSmb.java

```diff
@@ -184,10 +194,10 @@ public class ListSmb extends AbstractListProcessor<SmbListableEntity> {
             .build();
 
     private static final List<PropertyDescriptor> PROPERTIES = unmodifiableList(asList(
-            SMB_LISTING_STRATEGY,
             SMB_CLIENT_PROVIDER_SERVICE,
-            DIRECTORY,
             AbstractListProcessor.RECORD_WRITER,
+            SMB_LISTING_STRATEGY,
+            DIRECTORY,
```

Comment:
It is a good idea to move these properties after `SMB_CLIENT_PROVIDER_SERVICE` but `RECORD_WRITER` should not precedes them because it is not so important / not frequently used.
Please move `RECORD_WRITER` after `FILE_NAME_SUFFIX_FILTER`.